test-requirements.txt

```diff
@@ -1 +1,2 @@
 odoo-test-helper
+mock
```

Comment:
this is not needed. Just change the import of mock to `from unittest import mock` :wink:

Reply:
It **IS** needed because this depends on `storage_backend_sftp`, which also requires `mock`.
See https://github.com/OCA/connector-interfaces/actions/runs/10612682197/job/29414865086#step:8:209

Reply:
it is **NOT** because that's broken there too. It has been migrated in the wrong way. There's no need for such dep. Trust me :wink:

Reply:
in red the wrong ones 
![image](https://github.com/user-attachments/assets/c9576e38-2d86-4e3d-88a8-b2435d7978b4)

Reply:
There you go https://github.com/OCA/storage/pull/388